uiowa/uiowa
# Fix broken links in AOS migration.

Resolves #3275 

### Testing
- Go to https://admissions.dev.drupal.uiowa.edu/ and review AOS for broken links. Note that we are only fixing internal links. 
  -  https://admissions.dev.drupal.uiowa.edu/business is a good test case.
  - Other AOS can be found on https://admissions.dev.drupal.uiowa.edu/areas-of-study

Developer testing
```
blt sync --site admissions.uiowa.edu &&
drush @admissions.local entity:delete node --bundle area_of_study &&
drush @admissions.local entity:delete node --bundle transfer_tips &&
drush @admissions.local entity:delete taxonomy_term --bundle academic_groups &&
drush @admissions.local entity:delete taxonomy_term --bundle colleges &&
drush @admissions.local en admissions_migrate --yes &&
drush @admissions.local uli /admin/config/system/sitenow-migrate
```
- Configure for local DD admissions site.
- `drush @admissions.local ms`
- `drush @admissions.local mim --all`

Worth noting that running the migration again will attempt to fix broken links again which at best won't work or at worst replace them incorrectly. So after the step above:
- `drush @admissions.local pmu admissions_migrate`
- `drush @admissions.local cim`
